src/PrestaShopBundle/Kernel/ModuleRepository.php

```diff
@@ -69,7 +69,7 @@ public function getActiveModulesPaths()
     {
         $paths = array();
         $modulesFiles = Finder::create()->directories()->in(__DIR__.'/../../../modules')->depth(0);
-        $activeModules = array_keys($this->getActiveModules());
+        $activeModules = array_values($this->getActiveModules());
```

Comment:
`array_values` is maybe useless, only `in_array` is called.

Reply:
Yes, but `in_array` only check for values :)

Reply:
@mickaelandrieu array_values on your first example will return exactly the input ;)

Reply:
following should do the job
$activeModules = $this->getActiveModules();

Reply:
I hope you're right => update done.

Reply:
Yes were are right :)

```php
<?php

$array = [
    'foo',
    'bar',
    'baz',
];

$array2 = [
    'a' => 'foo',
    'b' => 'bar',
    'c' => 'baz',
];

var_dump(in_array('bar', $array));
var_dump(in_array('bar', $array2));
```

```bash
$ php test.php 
/home/got/test.php:15:
bool(true)
/home/got/test.php:16:
bool(true)
```

From documentation: 
> in_array — Checks if a value exists in an array

Reply:
I know, we can't troll in this team anymore :rofl:

Reply:
![https://www.bingeclock.com/memes/futurama___trolling_is_bad.jpg](https://www.bingeclock.com/memes/futurama___trolling_is_bad.jpg)